docs/source/getting-started/tabular-data/agg-and-filter.ipynb

```diff
@@ -0,0 +1,464 @@
+       "<Figure size 1200x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(12, 6))\n",
+    "sns.lineplot(x=count_year_all[\"Year-Quarter\"].to_list(), y=count_year_all['Actual Count'].to_list(), marker='o', label='Actual Count')\n",
+    "sns.lineplot(x=count_year_all[\"Year-Quarter\"].to_list(), y=count_year_all['DP Count'].to_list(), marker='o', label='DP Count')\n",
+    "\n",
+    "plt.title('Actual Count vs DP Count Over Years and Quarters')\n",
+    "plt.xlabel('Year-Quarter')\n",
+    "plt.ylabel('Count')\n",
+    "plt.xticks(rotation=45) \n",
+    "plt.legend()\n",
+    "\n",
+    "plt.grid(True)\n",
+    "plt.tight_layout()  \n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While the differentially private values still follow the general trend of the actual values, they do vary significantly more due to the smaller sizes of the year-quarter groups. \n",
+    "\n",
+    "### Caution Against Excessive Grouping\n",
+    "\n",
+    "Larger data generally corresponds to less noise being added and higher utility. Excessive grouping by multiple keys can lead to smaller partitions. The smaller size of each of the partitions may require more noise to be added to satisfy DP requirements. However, this dilutes the utility of the results. \n",
+    "\n",
+    "While grouping is an essential strategy for meaningful data analysis, there must be a balance excessive grouping can lead to noisy and misleading results."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filtering"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To demonstrate the `filter` method, compute the total number of respondents not working due to a layoff. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Actual Total Count of Laid Off Respondents:  76009\n",
+      "DP Total Count of Laid Off Respondents:  76251\n"
+     ]
+    }
+   ],
+   "source": [
+    "filt_actual = (\n",
+    "    df.filter(pl.col(\"ILOSTAT\")==3).select(\n",
+    "        pl.len()\n",
+    "    )\n",
+    ").collect().item()\n",
+    "print(\"Actual Total Count of Laid Off Respondents: \", filt_actual)\n",
+    "\n",
+    "filt_dp = (\n",
+    "    context.query().filter(pl.col(\"ILOSTAT\")==3).select(\n",
+    "        pl.len().dp.noise()\n",
+    "    )\n",
+    ").release().collect().item()\n",
+    "print(\"DP Total Count of Laid Off Respondents: \", filt_dp)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Filtering can also be viewed as a type of partitioning. If the result of filtering a smaller subset, it can lead to an increase in noise and decrease in utility. "
```

Comment:
The last sentence doesn't make much sense.

Reply:
what about this? 

Similar to partitioning, if the result of filtering is a smaller subset, it can lead to an increase in noise and decrease in utility.